test/examples/30_pybind11_examples/01_basics/02_object_oriented_code/chimera_pybind11.yaml

```diff
@@ -0,0 +1,13 @@
+# Arguments that will be appended in-order before command line arguments.
+arguments:
+  - "-extra-arg"
+  - "-I/usr/lib/clang/3.6/include"
+namespaces:
+  "chimera_test":
+    name: null # TODO: otherwise, import error
+types:
+  "const std::string &":
+    return_value_policy: copy
+classes:
+  "chimera_test::Pet":
+    dynamic_attr: True
```

Comment:
Open to better name for this.

Reply:
I think this is fine, since it mirrors the pybind11 interface and it's also only used in the pybind11 template now anyway.